react/javascript/demo-core-sdk/jest.config.js

```diff
@@ -24,10 +24,13 @@
 
  */
 
+const transformIgnorePatterns =
+  'node_modules/@looker/extension-sdk-react|node_modules/@looker/extension-sdk|node_modules/@looker/sdk'
```

Comment:
Not a big deal at all, but any reason for not using a regular array of regex strings rather than ORs in a single string?

Reply:
it's one eval vs. iterative evals?